gtfs/spec/en/reference.md

```diff
@@ -379,20 +379,25 @@ Shapes describe the physical path that a vehicle takes, and are defined in the f
 
 File: **Optional**
 
-This table is intended to represent schedules that don't have a fixed list of stop times. When trips are defined in frequencies.txt, the trip planner ignores the absolute values of the **arrival_time** and **departure_time** fields for those trips in [stop_times.txt](#stop_timestxt). Instead, the **stop_times** table defines the sequence of stops and the time difference between each stop.
+This table is used to represent two types of transit service:
+
+* frequency-based trips - Service that does not have a schedule of stop times (**exact_times** is **0** or **empty**).  Instead, operators make their best effort to serve stops periodically based on a headway.  For example, a rider can expect a bus to depart approximately every 15 minutes, but the vehicle will not attempt to adhere to a preset scheduled departure time such as 10:00 am.  Therefore, in the absence of real-time information, every subsequent departure is expected with a interval of **headway_secs** from the most recent departure. 
+* headway-defined schedules - Service that adheres to a schedule (**exact_times** is **1**).  This table can be used to represent simple scheduled service based on a fixed interval without fully listing out all scheduled trips via the [trips.txt](#tripstxt) and [stop_times.txt](#stop_timestxt) files.  Instead, the scheduled start time is defined in this table along with the set time between each departure (i.e., headway).  Consumers will then expand this to represent scheduled service as if each of these trips were defined in [trips.txt](#tripstxt) and [stop_times.txt](#stop_timestxt). 
+
```

Comment:
Yeah, I'm not in love with "headway-defined schedule" either - I'm open to proposals for how to name exact_times=1 service. "Headway" can definitely be associated with frequency-based service, but unfortunately we're stuck with the field name `headway_secs` for both types of service.  The main rule is that we can't use the word "frequency" for exact_times=1, and we can't use "schedule" for exact_times=0.